grocery_lists/api.py

```diff
@@ -0,0 +1,70 @@
+from grocery_lists.models import GroceryListCreate, GroceryListItemCreate, GroceryListUpdate
+from grocery_lists.services import add_item_to_grocery_list, create_grocery_list, delete_grocery_list, delete_grocery_list_item, get_grocery_list_items, get_grocery_lists_by_group_id, toggle_grocery_list_item, update_grocery_list
+from ninja import Router
+from django.http import JsonResponse
+
+router = Router()
+
+@router.get("/")
+def get_grocery_lists_handler(request, group_id: int):
+    try:
+        ret = get_grocery_lists_by_group_id(group_id)
+        return JsonResponse(ret, safe=False, status=200)
+    except Exception as e:
+        return JsonResponse({"status": "error", "message": "Error in fetching grocery lists"}, status=400)
```

Comment:
i think these should be 500s bc exception would be a server error (since you dont raise them if a list isnt found for example)